nix/devtools.nix

```diff
@@ -96,6 +97,29 @@ let
         ${style}/bin/postgrest-lint
         ${style}/bin/postgrest-style-check
       '';
+
+  importsgraph =
+    checkedShellScript
+      {
+        name = "postgrest-importsgraph";
+        docs =
+          ''
+            Render the imports between PostgREST modules as a graph.
+
+            Output is written to 'imports.png' in the root directory of the project.
+          '';
+        inRootDir = true;
+      }
+      ''
+        grep -rE 'import .*PostgREST\.' src \
+          | sed -E \
+              -e 's|/|\.|g' \
+              -e 's/src\.(.*)\.hs:import .*(PostgREST\.\S+)( .*)?/"\1" -> "\2"/' \
+              -e '1 i digraph \{' \
+              -e '$ a \}' \
+          | ${graphviz}/bin/dot -Tpng \
+          > imports.png
```

Comment:
Would it be possible to print the module Description(top comment of the file)? I wonder how would that look.

Reply:
Just added in a new commit - new output:

![imports](https://user-images.githubusercontent.com/59358383/107144494-c3915280-693b-11eb-81ed-f036e58fd9dd.png)